models/migrations/v1_23/v294.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+package v1_23 //nolint
+
+import (
+	"xorm.io/xorm"
+)
+
+// CommentMetaData stores metadata for a comment, these data will not be changed once inserted into database
+type CommentMetaData struct {
+	ProjectColumnID   int64  `json:"project_column_id"`
+	ProjectColumnName string `json:"project_column_name"`
+	ProjectName       string `json:"project_name"`
+}
```

Comment:
It seems that the field name is updated above but not be changed here?

Reply:
Fixed.

Reply:
You fixed `ProjectName`
But how about `ProjectColumnName` vs `ProjectColumnTitle` ?

Reply:
![image](https://github.com/go-gitea/gitea/assets/18380374/49bf5962-648c-4c77-8841-45fbcd6cd941)
![image](https://github.com/go-gitea/gitea/assets/18380374/719dae55-fc93-4ee7-8e8d-040d5234e1a2)